include/zephyr/bluetooth/mesh/keys.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_KEYS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_KEYS_H_
+
+#include <stdint.h>
+#if defined CONFIG_BT_MESH_USES_MBEDTLS_PSA
+#include <psa/crypto.h>
+#elif defined CONFIG_BT_MESH_USES_TINYCRYPT
+#include <string.h>
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum bt_mesh_key_type {
+	BT_MESH_ECB_KEY,
```

Comment:
I would really like to move `_KEY` to prefix:
`BT_MESH_KEY_<type>`, or even `BT_MESH_KEY_TYPE_<type>` (to follow enum name). This makes easier search for key usages in the code.